src/components/sections/Banner.tsx

```diff
@@ -1,7 +1,9 @@
 import React from "react";
-import { Link } from "react-router-dom";
-import ModalVideo from "react-modal-video";
-import "../../../node_modules/react-modal-video/scss/modal-video.scss";
+import hablmaos from "../../assets/images/hablamos-espanol.png";

```

Comment:
Just to keep it consistent, you should capitalize you import names just like `React`